src/pages/iou/WaypointEditor.js

```diff
@@ -144,6 +144,7 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
                 lat: null,
                 lng: null,
                 address: waypointValue,
+                name: waypointValue,
```

Comment:
We should set the name to `null` instead while offline. We only get the `name` field from the google auto-complete. It's redundant and confusing to duplicate the address as the name.